securedrop/bin/run

```diff
@@ -16,7 +16,19 @@ urandom
 build_redwood
 maybe_create_config_py
 run_redis
-reset_demo
+
+if [[ -e /var/lib/securedrop/db.sqlite ]]; then
+    echo "Using existing SecureDrop data..."
+    # Compile translated strings
+    pybabel compile --directory translations/
```

Comment:
Since it's just one line I think it's fine to duplicate and not refactor into a function.